python/lsst/ctrl/mpexec/dotTools.py

```diff
@@ -91,9 +108,10 @@ def _renderQuantumNode(
 
 def _renderDSTypeNode(name: str, dimensions: list[str], file: io.TextIOBase) -> None:
     """Render GV node for a dataset type"""
-    labels = [name]
+    pointSize = int(_ATTRIBS["defaultNode"].get("fontsize", 12)) + 4
```

Comment:
Hard-coded values are usually not welcome, could you add "fontsize" attribute to `_ATTRIBS["task"]` instead?

Reply:
There already is a `fontsize` attribute being set in `defaultNode`, which will be applied to all nodes (including task nodes). The line you've highlighted here is a safety net that will try to get `fontsize` from `defaultNode`, but, if someone were to remove `fontsize` down the line for some reason, it will fall back to a default value of 12.

Reply:
If you're not happy with setting this default here, I can remove the `12` and simply allow the code to fail at some future date if/when someone were to remove the `fontsize` attribute from `defaultNode`. Perhaps failing then is safer in any case?

Reply:
I was mostly thinking about `+ 4`, but now I see that `fontsize` to task attributes also changes size of all labels, while intent here is to change the size of task name only. Maybe a separate "task-name" key can be added to _ATTRIBS to hold that value? Default `12` does not do much here, indeed, may be worth dropping it or defining a separate module-level constant for it instead?

Reply:
How about I replace this logic entirely with an externally-set `_NODELABELPOINTSIZE` attribute, set to 18? It's not just tasks that are impacted, it also impacts dataset type label names too. The intent behind the `+4` was that, if the user did change `fontsize`, the labels would still be somewhat larger. However, I'm happy to forego that convenience if hardcoding it makes it easier to read/maintain?